engine/src/main/java/org/hibernate/search/engine/backend/metamodel/IndexCompositeElementDescriptor.java

```diff
@@ -44,8 +45,19 @@
 	 * fields created dynamically through {@link IndexSchemaElement#fieldTemplate(String, Function) templates}
 	 * are not included in the collection.
 	 *
-	 * @return A collection containing all fields.
+	 * @return A collection containing all static child fields.
 	 */
 	Collection<? extends IndexFieldDescriptor> staticChildren();
 
+	/**
+	 * Get all statically-defined, direct child fields for this element, mapped by their relative name.
```

Comment:
Is relative name `IndexFieldDescriptor#relativeName`? Is would be not critical, but if so, maybe we could add the link to the method in the Javadoc.

Reply:
Good idea. Done.